pkg/datapath/linux/route/route_linux_test.go

```diff
@@ -279,6 +279,16 @@ func runListRules(c *C, family int, fakeIP, fakeIP2 *net.IPNet) {
 
 	defaultRules, _ := ListRules(family, nil)
 
+	ruleMinus1 := netlink.Rule{
+		Priority: -1, Mark: -1, Mask: -1, Goto: -1, Table: 255, Flow: -1, SuppressIfgroup: -1, SuppressPrefixlen: -1,
+	}
+	rule32766 := netlink.Rule{
+		Priority: 32766, Mark: -1, Mask: -1, Goto: -1, Table: 254, Flow: -1, SuppressIfgroup: -1, SuppressPrefixlen: -1,
+	}
+	rule32767 := netlink.Rule{
+		Priority: 32767, Mark: -1, Mask: -1, Goto: -1, Table: 253, Flow: -1, SuppressIfgroup: -1, SuppressPrefixlen: -1,
+	}
```

Comment:
Do we know why we need these now? The original function should have been equivalent to the upstream function that we're not using.

Reply:
I did not set the mask for the filter correctly. That's why these additional rules were returned. Fix it now.